File: src/main/java/com/itis/timetable/data/models/schedule/ScheduleEntity.java
package com.itis.timetable.data.models.schedule;

import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;
import lombok.ToString;

import javax.persistence.Column;
import javax.persistence.Entity;
import javax.persistence.Id;
import javax.persistence.Table;

@NoArgsConstructor
@Getter
@Setter
@ToString
@Entity
@Table(name = "schedules")
public class ScheduleEntity {
    @Id
    private long id;
    @Column(name = "group_id")
    private long groupId;

    public ScheduleEntity(long id, long groupId) {
        this.id = id;
        this.groupId = groupId;
    }
}

File: src/main/kotlin/com/itis/timetable/data/models/schedule/ScheduleEntity.kt
package com.itis.timetable.data.models.schedule

/*
@Entity(
    tableName = "schedules",
    foreignKeys = [
        ForeignKey(
            entity = Group::class,
            parentColumns = ["id"],
            childColumns = ["group_id"]
        )
    ],
)*/
data class ScheduleEntity(
    val id: Long,
    val groupId: Long,
)

